xcode/Subconscious/Shared/Components/AddressBook/AddressBookView.swift

```diff
@@ -92,6 +80,28 @@ struct AddressBookView: View {
             ) {
                 FollowUserView(state: state, send: send)
             }
+            .alert(
+                state.failUnfollowErrorMessage ?? "",
+                isPresented: Binding(
+                    get: { state.failUnfollowErrorMessage != nil },
+                    set: { _ in send(.dismissFailUnfollowError) }
+                )
+            ) {}
```

Comment:
Q: is this alert doing anything? The view portion of the alert seems to be empty.

Reply:
This works fine (renders a simple message + OK button) but I found a better function signature for the job. We can present both a title and message this way too: 

<img width="302" alt="image" src="https://user-images.githubusercontent.com/5009316/226496777-0f642d58-33be-4758-b103-531719eb7ece.png">